tests/conftest.py

```diff
@@ -0,0 +1,71 @@
+from pathlib import Path
+from unittest import mock
+
+import pytest
+import yaml
+from prefect.blocks.kubernetes import KubernetesClusterConfig
+
+from prefect_kubernetes.credentials import KubernetesCredentials
+
+BASEDIR = Path("tests")
+GOOD_CONFIG_FILE_PATH = BASEDIR / "kube_config.yaml"
+
+
+@pytest.fixture
+def kube_config_dict():
+    return yaml.safe_load(GOOD_CONFIG_FILE_PATH.read_text())
+
+
+@pytest.fixture
+def successful_job_status():
+    job_status = mock.MagicMock()
+    job_status.status.active = None
+    job_status.status.failed = None
+    job_status.status.succeeded = 1
+    return job_status
+
+
+@pytest.fixture
+def kubernetes_credentials(kube_config_dict):
+
+    return KubernetesCredentials(
+        cluster_config=KubernetesClusterConfig(
+            context_name="test", config=kube_config_dict
+        )
+    )
+
+
+@pytest.fixture
+def api_app_client(monkeypatch):
```

Comment:
I like to name these fixtures suffixed with `_mock` so developers know that it's magic mock